### PR TITLE
Add onnx support for `torch.lerp`

### LINKED ISSUE #68384: RuntimeError: Exporting the operator lerp to ONNX opset version 12 is not supported.
## 🚀 Feature
Support for exporting the `lerp` operator to ONNX.

## Motivation
I'm trying to export the generator of StyleGAN to ONNX, and am getting this error (no matter the opset version):
```
RuntimeError: Exporting the operator lerp to ONNX opset version 12 is not supported. Please feel free to request support or submit a pull request on PyTorch GitHub.
```

Here's a Colab reproducing the error:
* https://colab.research.google.com/gist/josephrocca/b8fc780ac8e04408b69cce361a16a4f3/stylegan3-to-onnx-conversion.ipynb
* https://gist.github.com/josephrocca/b8fc780ac8e04408b69cce361a16a4f3

Thanks!

### PR DESCRIPTION
**Pull request by qqaatw**
Fixes #68384

### CODE REVIEW

File: torch/onnx/symbolic_opset9.py [hunk 1]
```diff
@@ -4892,6 +4892,26 @@ def cdist(g, x1, x2, p=2.0, compute_mode="use_mm_for_euclid_dist_if_necessary"):
...
+        g,
+        g.op("Less", weight, g.op("Constant", value_t=torch.tensor(0.5))),
+        g.op("Add", self, g.op("Mul", weight, diff)),
+        g.op(
+            "Sub",
+            end,
+            g.op(
+                "Mul",
+                diff,
+                g.op("Sub", g.op("Constant", value_t=torch.tensor(1)), weight),
```
Comment (BowenBao) [inline torch/onnx/symbolic_opset9.py:4909]: nit: dtype consistency
```suggestion
                g.op("Sub", g.op("Constant", value_t=torch.tensor(1.0)), weight),
```

**Review by BowenBao — APPROVED** [review]
LGTM, thank you @qqaatw for contributing!

**Comment by qqaatw** [comment]
Looks that there is a memory issue with `TestONNXRuntime_opset9.test_groupnorm`, the last CI check returned `137` (OOM) error code.

**Pull request closed without merging**